cypress.config.js

```diff
@@ -2,7 +2,7 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
-    setupNodeEvents(on, config) {
-    },
+    baseUrl: "https://demoqa.com",
+    
```

Comment:
There is an unnecessary trailing whitespace at the end of the line. It is a good practice to remove trailing whitespaces to maintain a clean codebase. 🧹